Core/Lib/Export/PDFExport.php

```diff
@@ -264,20 +272,26 @@ protected function insertBusinessDocFooter($model)
             'currency' => $this->i18n->trans('currency'),
             'net' => $this->i18n->trans('net'),
             'taxes' => $this->i18n->trans('taxes'),
+            'totalSurcharge' => $this->i18n->trans('total') . ' - ' . $this->i18n->trans('surcharge'),
```

Comment:
'totalsSurcharge' => $this->i18n->trans('surcharge'),
'totalIrpf' => $this->i18n->trans('irpf'),